test/CMakeLists.txt

```diff
@@ -1,3 +1,4 @@
 add_subdirectory(utils)
 add_subdirectory(bench)
-add_subdirectory(unittests)
\ No newline at end of file
+add_subdirectory(unittests)
+add_subdirectory(spectests)
```

Comment:
I put `spectests` before `unittests` and didn't move `utils`, because supposedly `bench` depends on it. But actually putting `utils` on the last line works for me, too.